src/components/LoginView/LoginView.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (event) => {
+    // this prevents the default behavior of the form, to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      access: username,
+      secret: password
+    };
+
+    fetch("https://cf-movie-api.herokuapp.com/login", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    })
+    .then((response) => response.json())
+    .then((data) => {
+      console.log("Login response: ", data);
```

Comment:
Always remove console logs when pushing or merging to the main branch

Reply:
Thank you removed.